modules/nf-core/nanoplot/tests/main.nf.test

```diff
@@ -63,7 +63,41 @@ nextflow_process {
                     with(process.out.html.get(0)) {
                         assert get(1).collect { p -> file(p).getName() }.contains("NanoPlot-report.html")
                     }
-                } 
+                }
+            )
+        }
+
+    }
+
+    test("NanoPlot - stub") {
+
+        options "-stub"
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    [ file(params.test_data['sarscov2']['nanopore']['test_sequencing_summary'], checkIfExists: true) ]
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                            process.out.txt,
+                            process.out.html,
+                            process.out.log,
+                            process.out.versions
+                         ).match()
```

Comment:
```suggestion
                { assert snapshot(process.out).match()
```
why not like this?

Reply:
Does it matter that `process.out.png` is empty (well optional and empty)?

Reply:
not for stubs